protocol/migrations/sqlite/1704489636_add_album_images.up.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE user_messages ADD COLUMN album_images TEXT DEFAULT "";
```

Comment:
should it be stored as BLOB?

Reply:
BLOB are best used for storing binary data which we are not storing here.

Reply:
oh, so it is not an image content, right?

Reply:
> oh, so it is not an image content, right?

It's kind of an image URL served. I chose that column name because it's better for understanding as I don't want to expose the internal implementation which may change in the future?